README.md

```diff
@@ -80,14 +106,14 @@ kubectl apply -f $PLUGIN_TEMPLATE
 ```
 
 ```yaml
-# .drone.yml
+# .woodpecker.yml
 pipeline:
   deploy-to-k8s:
     image: laszlocloud/my-k8s-plugin
     template: config/k8s/service.yml
 ```
 
-## Documentation
+# Documentation
```

Comment:
```suggestion
## Documentation
```

As it is a sub-heading of Woodpecker I think it should be `##`, but maybe we can move the docs link a bit to the top?